boards/px4/fmu-v6x/init/rc.board_defaults

```diff
@@ -13,6 +13,9 @@ then
 	param set SENS_MAG_MODE 0
 
 	param set UAVCAN_ENABLE 2
+
+	# Use always FMU as RC input
+	param set SYS_RC_SOURCE 2
```

Comment:
This is confusing "Enable FMU RC output as new default" says "# Use always FMU as RC input"

Also it is not supported by the HW

Reply:
It will need to support it because FMUv6X will largely ship without an IO board - if that statement is overall true then the hardware is wrong.

Reply:
I guess what you're saying is not that the hardware doesn't support it. What you're saying is that only FMU depends on the board build config (in contrast to FMUv5, which wires it up to both processors). That means we would rely on the IO detection instead.